generate_build_config.py

```diff
@@ -45,8 +46,45 @@
   permissions: '0755'
 """
 
+PRIVATE_PPA_TEMPLATE = """
+- chroot $CHROOT_ROOT apt install apt-transport-https
```

Comment:
Good point.

It does, indeed. The reason I didn't notice is that it's already installed in the build environment. While that means it's not strictly necessary I think it's better to leave it (I'll update the branch to add a -y).